styles.css

```diff
@@ -0,0 +1,1118 @@
+  position: fixed;
+  z-index: 100;
+  padding-top: 50px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(96, 112, 255);
+  background-color: rgb(96, 112, 255, 0.95);
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(15px);
+}
+
+.btn-wrapper {
+  position: fixed;
+  top: 10px;
+  left: 10px;
+}
+
+.close_btn {
+  border: none;
+  cursor: pointer;
+  width: 45px;
+  outline: none;
+  background: none;
+}
+
+.desktop_nav {
+  display: none;
+}
+
+.nav_items {
+  font-size: 3em;
+  align-items: center;
+  margin-left: 50px;
+  padding: 20px;
+}
+
+a.email {
+  display: block;
+  font-size: 0.8em;
+  font-weight: 700;
+  text-underline-position: under;
+  line-height: 1.5rem;
+  color: #272a31;
+}
+
+#error {
+  color: red;
+  width: 300px;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.course_lecturers {
+  margin-left: 10px;
+  margin-top: 14px;
+}
+
+.checkers {
+  width: 40px;
+  position: relative;
+}
+
+.lecturer_content {
+  margin-left: 100px;
+}
+
+.hidden_features {
+  display: none;
+}
+
+.dynamic {
+  display: flex;
+}
+
+#lecturer-list {
+  display: flex;
+  flex-direction: column;
+  margin-top: 1.5rem;
+  gap: 2rem;
+}
+
+#lecturer-list li {
+  display: flex;
+  gap: 1.5rem;
+}
+
+#lecturer-list li:nth-child(n+3) {
+  display: none;
+}
+
+.lecturer-details h3 {
+  font-size: 1rem;
+}
+
+.lecturer-image {
+  background-image: url('images/image-bg.png');
+  background-size: contain;
+  background-position: top left;
+  background-repeat: no-repeat;
+  max-width: 100px;
+  flex: 1;
+}
+
+.lecturer-image img {
+  position: relative;
+  width: 70px;
+  height: 70px;
+  top: 0.73rem;
+  left: 0.8rem;
+}
+
+.lecturer-details {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  flex: 2.5;
+}
+
+.lecturer-title {
+  font-size: 0.82rem;
+  font-weight: 600;
+  line-height: 1.2rem;
+  color: #ec5242;
+}
+
+hr.grey-line {
+  background-color: #d3d3d3;
+  border: #d3d3d3;
+  height: 0.1rem;
+  width: 5%;
+}
+
+.lecturer-info {
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+  margin-bottom: 10px;
+}
+
+#hr-hero {
+  display: none;
+}
+
+#arrow {
+  display: none;
+}
+
+.about-partner {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+a.link,
+a.linkA,
+a.linkB {
+  color: white;
+  text-decoration: none;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-bottom: 30px;
+}
+
+.feat-heading {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.top-nav {
+  display: none;
+}
+
+/* ----MEDIA QUERIES SECTION---- */
+
+/* || DESKTOP VERSION */
+@media (min-width: 768px) {
+  .header_items {
+    display: flex;
+    padding: 5px 5px;
+    gap: 6%;
+    align-items: center;
+    flex-direction: row;
+    margin-left: 900px;
+  }
+
+  .desktop_nav {
+    display: inline-flex;
+    row-gap: 5%;
+    width: 100%;
+  }
```

Comment:
- [ ] [OPTIONAL] The navigation bar alignment is still looking a bit off, maybe it is happening because we have different screen sizes. I experimented a little and found better results when adding the following rules on `.desktop_nav`, kindly give it a try. 🤝️

```
  justify-content: space-between;
  padding: 0 120px;
```